src/components/actions/resumeProcessing.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * If the server is shut down while there are still triggers being processed in
+ * the "Throttle" queue, this will ensure that they get re-started. This should
+ * ONLY be called at start-up, before any other trigger/action related methods.
+ */
```

Comment:
This shouldn't be needed in normal functioning, but if the server *does* happen to get shut down (or crash) during a large trigger batch process, then incomplete triggers would be lost and never complete.